tests/e2e/testfiles/values.yaml

```diff
@@ -910,6 +910,128 @@
     - "Enable forceRemoveDriver on CR [1]"
     - "Delete custom resource [1]"
 
+- scenario: "Install PowerFlex Driver(With Resiliency)"
+  paths:
+    - "testfiles/storage_csm_powerflex_resiliency.yaml"
+  modules:
+    - "resiliency"
+  steps:
+    - "Given an environment with k8s or openshift, and CSM operator installed"
+    - "Apply custom resource [1]"
+    - "Validate custom resource [1]"
+    - "Validate [powerflex] driver from CR [1] is installed"
+    - "Validate [resiliency] module from CR [1] is installed"
+    - "Run custom test"
+    # Last two steps perform Clean Up
+    - "Enable forceRemoveDriver on CR [1]"
+    - "Delete custom resource [1]"
+  customTest:
+    name: HelloWorld 
+    run:
+     - "echo Hello && echo World"
```

Comment:
can we run cert-csi/resiliency e2e to make sure driver+resiliency is working?

Reply:
We have ran it.

Reply:
I'm asking if we can run that for customTest in the nightly jobs.